### ISSUE TITLE
Compiler error messages should not be cropped

### ISSUE THREAD

**Issue by opyh** [spark/spark-dev#77]
Compiler error messages are cropped in the UI:

![screenshot 2015-03-22 03 51 44](https://cloud.githubusercontent.com/assets/192912/6767828/06e2004a-d048-11e4-9ad2-c960fc79a7a4.png)

Have I overlooked a way to see the full error messages somewhere? That would be helpful when debugging, as the important bits are often at the (cut-off) ends of the messages.

Maybe they could just be logged to the console?

Even better would be having inline error messages, like in the [linter plugin](https://atom.io/packages/linter) ;)

**Comment by rsteckler** [comment]
Have you found a way to work around this?  I'm having a hard time determinging what the compiler is complaining about.

**Comment by suda** [comment]
This will be probably done by implementing #79 or some logging panel. Currently the main goal is updating to latest Atom and fully support upcoming Photon.

**Comment by cdeadlock** [comment]
I had to do this pain in the arse to see full errors for another problem #110 

In atom go to the menu -> View -> Developer -> toggle developer tools
Its just like chrome or chromium developer tools: you can click on the magnifying glass and click on a UI element, and see the source for the full information

**Comment by newzealandpaul** [comment]
This is a real pain. Is this being worked on?

**Comment by suda** [comment]
Yes, we're going to introduce error console similar to one in Build IDE.

**Comment by ponuganti** [comment]
This is rea big pain.. I get the error "build didn't produce binary Error: Command faild:.." I could see the full error .. this is minimum is ide that it shows what the error is ..

**Comment by dhhagan** [comment]
Has this been implemented yet? This is the least lovable Particle Dev feature right now..

**Comment by suda** [comment]
Unfortunately not yet. But it is on top of your Dev priorities!
